tests/translator/input/api_with_resource_policy.yaml

```diff
@@ -0,0 +1,12 @@
+Resources:
+ ExplicitApi:
+    Type: AWS::Serverless::Api
+    Properties:
+      StageName: Prod
+      Auth:
+        ResourcePolicy:
+          CustomStatements: [{
+              Action: 'execute-api:Invoke', # Optional; Default: execute-api:Invoke
```

Comment:
This isn't currently optional, correct? Making this optional would happen in a future PR.

Reply:
will remove. thanks!